src/libraries/System.Console/src/System/TermInfo.Database.cs

```diff
@@ -75,12 +75,14 @@ internal Database(string term, byte[] data)
             // (Note that the extended section also includes other Booleans and numbers, but we don't
             // have any need for those now, so we don't parse them.)
             int extendedBeginning = RoundUpToEven(StringsTableOffset + _stringTableNumBytes);
-            _extendedStrings = ParseExtendedStrings(data, extendedBeginning, _readAs32Bit) ?? new Dictionary<string, string>();
+            _extendedStrings = ParseExtendedStrings(data, extendedBeginning, _readAs32Bit);
         }
 
         /// <summary>The name of the associated terminfo, if any.</summary>
         public string Term { get { return _term; } }
 
+        internal bool HasExtendedStrings => _extendedStrings is not null;
```

Comment:
Alternatively you could just make the _extendedStrings dictionary public as ExtendedStrings.